src/tokens/polygonTokens.json

```diff
@@ -20518,5 +20518,23 @@
                 "website": "https://peercoin.net/"
             }
         }
+    },
+    {
+        "chainId": 137,
+        "name": "Giveth Token",
+        "symbol": "GIV",
+        "decimals": 18,
+        "address": "0xc4df120d75604307dcB604fde2AD3b8a8B7c6FAA",
+        "logoURI": "https://github.com/Giveth/giveth-design-assets/blob/master/02-logos/GIV%20Token/GIVToken_256x256.png",
+        "tags": ["pos", "erc20","swapable"],
```

Comment:
This tokens seems to be mapped over fx bridge. Please remove the pos tag and add the fx tag.